models/__tests__/categories.test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
```

Comment:
Good start on categories, looks like you are still working on products. It should be very similar with a different schema.